access/slack/app.go

```diff
@@ -190,6 +191,64 @@ func (a *App) onWatcherEvent(ctx context.Context, event access.Event) error {
 	}
 }
 
+func (a *App) processSlackCallback(ctx context.Context, req access.Request, cb Callback) (RequestData, string, error) {
+	action := cb.ActionCallback.BlockActions[0]
+	actionID := action.ActionID
+
+	reqData := RequestData{User: req.User, Roles: req.Roles, RequestReason: req.RequestReason}
+	// already handled elsewhere
```

Comment:
whats already handled elsewhere?

Reply:
it means that the request was already approved/denied somewhere else